source/frontend/src/models/generate/acquisition/GenerateInterestHolder.ts

```diff
@@ -1,26 +1,37 @@
-import { Api_InterestHolder, Api_InterestHolderProperty } from '@/models/api/InterestHolder';
-import Api_TypeCode from '@/models/api/TypeCode';
+import { getApiPersonOrOrgMailingAddress } from '@/features/contacts/contactUtils';
+import { Api_InterestHolder } from '@/models/api/InterestHolder';
 import { formatNames } from '@/utils/personUtils';
 
-export class Api_GenerateInterestHolder {
-  interestHolderName: string;
-  interestHolderType: string;
-  interestHolderString: string;
+import { Api_GenerateAddress } from '../GenerateAddress';
+import { Api_GeneratePerson } from '../GeneratePerson';
```

Comment:
This file was renamed as-is from Api_GenerateExpropriationInterestHolder - because this model represents the actual interest holder and its relationships as they are returned from the API